iron-image.html

```diff
@@ -77,6 +79,12 @@
       position: relative;
     }
 
+    #img {
+      display: block;
```

Comment:
if we do this, the img will always take 100% of the container, and the iron-image size would become larger than needed.
Should this be inline-block instead?